docstore/doc.go

```diff
@@ -231,8 +231,8 @@
 //   iter := coll.Query().Where("Author.Last", "=", "Bulgakov").Limit(3).Get(ctx)
 //
 // You can make multiple Where calls. In some cases, parts of a Where clause may be
-// processed on the client rather than natively by the provider, which may have
-// performance implications for large result sets. See the provider-specific package
+// processed in the driver rather than natively by the backend service, which may have
```

Comment:
I believe "backing service" is more common with cloud/apps.

Reply:
Done.